pymc3/distributions/discrete.py

```diff
@@ -823,21 +899,22 @@ class DiscreteUniform(Discrete):
 
     .. plot::
 
-        import matplotlib.pyplot as plt
-        import numpy as np
-        import scipy.stats as st
-        plt.style.use('seaborn-darkgrid')
-        ls = [1, -2]
-        us = [6, 2]
-        for l, u in zip(ls, us):
-            x = np.arange(l, u+1)
-            pmf = [1.0 / (u - l + 1)] * len(x)
-            plt.plot(x, pmf, '-o', label='lower = {}, upper = {}'.format(l, u))
-        plt.xlabel('x', fontsize=12)
-        plt.ylabel('f(x)', fontsize=12)
-        plt.ylim(0, 0.4)
-        plt.legend(loc=1)
-        plt.show()
+		import matplotlib.pyplot as plt
```

Comment:
These and the below shouldn't show up, or were these broken?

Reply:
Oh damn sorry, yeah you're right. Messed that up, will change it.